components/search/MoreFiltersDropdown.vue

```diff
@@ -148,6 +154,21 @@
         }
         return filtersChanged;
       },
+      filterMoreFacets() {
```

Comment:
Much of the additional facet-specific logic in the components can be removed by making it the responsibility of the store instead, thus:
```diff
diff --git a/components/search/MoreFiltersDropdown.vue b/components/search/MoreFiltersDropdown.vue
index 44d9f1e7..77f876f9 100644
--- a/components/search/MoreFiltersDropdown.vue
+++ b/components/search/MoreFiltersDropdown.vue
@@ -38,7 +38,7 @@
           />
         </template>
         <template
-          v-for="(facet, index) in filterMoreFacets"
+          v-for="(facet, index) in moreFacets"
         >
           <MoreFiltersDropdownFacet
             v-if="facet.fields && facet.fields.length > 0"
@@ -107,10 +107,6 @@
       selected: {
         type: Object,
         default: () => {}
-      },
-      showContentTierToggle: {
-        type: Boolean,
-        default: true
       }
     },
     data() {
@@ -154,21 +150,6 @@
         }
         return filtersChanged;
       },
-      filterMoreFacets() {
-        const filtered = [];
-        for (const facet of this.moreFacets) {
-          // decide whether or not to show the contenttier
-          // TODO: remove tierToggleEnabled check
-          if (facet.name === 'contentTier') {
-            if (this.tierToggleEnabled && this.showContentTierToggle) {
-              filtered.push(facet);
-            }
-          } else {
-            filtered.push(facet);
-          }
-        }
-        return filtered;
-      },
       moreFacetNames() {
         return this.moreFacets.map((facet) => facet.name);
       },
@@ -182,10 +163,6 @@
           return { start: proxyDctermsIssued[0], end: null, specific: true };
         }
         return range;
-      },
-      // TODO: remove tierToggleEnabled check
-      tierToggleEnabled() {
-        return Boolean(Number(process.env['ENABLE_CONTENT_TIER_TOGGLE']));
       }
     },
     watch: {
diff --git a/components/search/SearchInterface.vue b/components/search/SearchInterface.vue
index 09e43ed6..00625a1d 100644
--- a/components/search/SearchInterface.vue
+++ b/components/search/SearchInterface.vue
@@ -30,7 +30,6 @@
             />
             <MoreFiltersDropdown
               v-if="enableMoreFacets"
-              :show-content-tier-toggle="hasContentTierToggle"
               :more-facets="moreFacets"
               :selected="moreSelectedFacets"
               @changed="changeMoreFacets"
@@ -168,8 +167,7 @@
     data() {
       return {
         coreFacetNames: ['collection', 'TYPE', 'COUNTRY', 'REUSABILITY'],
-        PROXY_DCTERMS_ISSUED: 'proxy_dcterms_issued',
-        hasContentTierToggle: this.showContentTierToggle
+        PROXY_DCTERMS_ISSUED: 'proxy_dcterms_issued'
       };
     },
     computed: {
diff --git a/store/search.js b/store/search.js
index 840d5b96..984a9f5f 100644
--- a/store/search.js
+++ b/store/search.js
@@ -257,7 +257,7 @@ export const actions = {
   },
 
   // TODO: replace with a getter?
-  async deriveApiSettings({ commit, dispatch, state }) {
+  async deriveApiSettings({ commit, dispatch, state, getters }) {
     // Coerce qf from user input into an array as it may be a single string
     const userParams = Object.assign({}, state.userParams || {});
     userParams.qf = [].concat(userParams.qf || []);
@@ -281,11 +281,21 @@ export const actions = {
     commit('set', ['apiParams', apiParams]);
     commit('set', ['apiOptions', apiOptions]);
 
-    await dispatch('applyCollectionSpecificSettings');
+    if (getters.collection) {
+      await dispatch('applyAnyCollectionSettings');
+      await dispatch('applyCollectionSpecificSettings');
+    }
+  },
+
+  applyAnyCollectionSettings({ commit, state }) {
+    const facet = state.apiParams.facet.split(',');
+    facet.splice(facet.indexOf('contentTier'), 1);
+    commit('set', ['apiParams', { ...state.apiParams, ...{ facet: facet.join(',') } }]);
   },
 
   applyCollectionSpecificSettings({ commit, getters, rootGetters, rootState, state }) {
     const collection = getters.collection;
+
     if (!getters.hasCollectionSpecificSettings(collection)) return;
 
     for (const property of ['apiParams', 'apiOptions']) {
```